src/Peak_Tracking.py

```diff
@@ -24,14 +27,20 @@
 p1 = 0 #p is index in position array
 p2 = 0
 
+def find_next_peak_index(arr, indx):
+"""Find the next index of a peak in the array"""
+    while indx < len(arr) - 1:
+        if arr[idnx] <= arr[indx+1]
+            return indx
+        idnx += 1
+    return indx
```

Comment:
Great use of a function here, but potentially unclear as to what arguments are being passed and returned.  Consider making variable names more specific and adding descriptions of arguments in the comment block (see above).